tests/test_render.py

```diff
@@ -486,10 +487,13 @@ def update_output(value):
         self.percy_snapshot(name='simple-callback-1')
 
         input1 = self.wait_for_element_by_css_selector('#input')
-        input1.send_keys(Keys.CONTROL + "a")
-        input1.send_keys(Keys.DELETE);
+        initialValue = input1.get_attribute('value')
 
-        input1.send_keys('hello world')
+        action = ActionChains(self.driver)
+        action.click(input1)
+        action = action.send_keys(Keys.BACKSPACE * len(initialValue))
```

Comment:
Ha ok, super annoying but you're right, neither `META` nor `COMMAND` actually seems to work on a mac. At some point we should probably make a util for clearing an input 🌴 but this is good for now.